ttnn-visualizer/src/scss/_colorSchemeLight.scss

```diff
@@ -0,0 +1,18 @@
+// SPDX-License-Identifier: Apache-2.0
+//
+// SPDX-FileCopyrightText: © 2024 Tenstorrent Inc.
+// Unusable as it is
+@media (prefers-color-scheme: light) {
```

Comment:
excellent but the rest of the app should respond to color scheme as well. we cant only control bg color. i suggest stickign to theme dark for now and potentially resurection this in the future as a single improvement feature

Reply:
Commented this all out but added a todo.